test/functional/p2p_reqtxrcncl.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+# Copyright (c) 2023 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test REQTXRCNCL message
+"""
+
+import time
+
+from test_framework.p2p import P2PInterface
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal
+from test_framework.wallet import MiniWallet
+
+class ReqTxrcnclReceiver(P2PInterface):
+    def __init__(self):
+        super().__init__(support_txrcncl = True)
+        self.reqtxrcncl_msg_received = None
+        self.received_inv_items = 0
+
+    def on_inv(self, message):
+        self.received_inv_items += len(message.inv)
+        super().on_inv(message)
+
+    def on_reqtxrcncl(self, message):
+        self.reqtxrcncl_msg_received = message
+
+class ReqTxRcnclTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.num_nodes = 1
+        self.extra_args = [['-txreconciliation']]
+
+    def run_test(self):
+        t = int(time.time())
+        self.nodes[0].setmocktime(t)
+        self.generate(self.nodes[0], 200) # mature coinbase UTXO used later
```

Comment:
If the purpose is maturing them, I think we could do:
```suggestion
        self.generate(self.nodes[0], COINBASE_MATURITY) # mature coinbase UTXO used later
```